gen3/bin/psql.sh

```diff
@@ -58,11 +51,14 @@ g3k_psql() {
     return 1
     ;;
   esac
-  local username=$(jq -r ".${key}.db_username" < $credsPath)
-  local password=$(jq -r ".${key}.db_password" < $credsPath)
-  local host=$(jq -r ".${key}.db_host" < $credsPath)
-  local database=$(jq -r ".${key}.db_database" < $credsPath)
-
+  local credsPath
+  credsPath="$(mktemp "${XDG_RUNTIME_DIR}/creds.json.XXXXXX")"
+  g3kubectl get secret "${key}-creds" -o json | jq -r '.data["creds.json"]' | base64 --decode > "$credsPath"
+  local username=$(jq -r ".db_username" < $credsPath)
+  local password=$(jq -r ".db_password" < $credsPath)
+  local host=$(jq -r ".db_host" < $credsPath)
+  local database=$(jq -r ".db_database" < $credsPath)
+  rm "$credsPath"
```

Comment:
For max security why don't we shred the file before deleting it `shred ${credsPath}`